core/app.go

```diff
@@ -102,6 +103,15 @@ func NewApp(configFlag string) (*App, error) {
 	a.Tasks = cfg.Tasks
 	a.Telemetry = cfg.Telemetry
 	a.ConfigFlag = configFlag
+
+	// set an environment variable for each service IP address so that
+	// forked processes have access to this information
+	for _, service := range a.Services {
+		envKey := fmt.Sprintf("CONTAINERPILOT_%v_IP",
+			strings.ToUpper(service.Name))
```

Comment:
Unsure, but do we have to sanitize this? ie: converting dashes to underscores, removing characters that shouldn't be part of an environment variable.